src/main/java/net/kaczmarzyk/spring/data/jpa/web/SpecificationArgumentResolver.java

```diff
@@ -41,12 +42,18 @@
 	private static Map<Class<? extends Annotation>, SpecificationResolver<? extends Annotation>> resolversBySupportedType;
 
 	public SpecificationArgumentResolver() {
+		 this(null);
+	}
+	
+	public SpecificationArgumentResolver(ConversionService conversionService) {
+		SimpleSpecificationResolver simpleSpecificationResolver = new SimpleSpecificationResolver(conversionService);
+		
 		resolversBySupportedType = Arrays.asList(
-				new SimpleSpecificationResolver(),
-				new OrSpecificationResolver(),
-				new DisjunctionSpecificationResolver(),
-				new ConjunctionSpecificationResolver(),
-				new AndSpecificationResolver(),
+				simpleSpecificationResolver,
```

Comment:
please double-check that simpleSpecificationResolver is threadsafe (i.e. it does not have any state (mutable fields))